python-package/xgboost/spark.py

```diff
@@ -0,0 +1,419 @@
+#
+# Copyright (c) 2019 by Contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import re
+
+from pyspark.ml.param import Params
+from pyspark.ml.util import JavaMLWritable
+from pyspark.ml.wrapper import JavaModel, JavaEstimator
+
+from sparkxgb.util import XGBoostReadable
```

Comment:
it appears that this import was left in unintentionally? I'm assuming that's causing Ben's import issue (as you define the class later on as well).

Is the new import simply `from xgboost.spark import XGBoostRegressor` rather than `from sparkxgb import XGBoostRegressor` with this change?